test/Sentry.Serilog.Tests/Sentry.Serilog.Tests.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net462</TargetFrameworks>
```

Comment:
@bruno-garcia the other test projects, except for the Log4Net one, target `netcoreapp2.1;netcoreapp2.0;net462`. Should these projects do the same?